tests/rptest/redpanda_cloud_tests/config_profile_verify_test.py

```diff
@@ -71,6 +75,52 @@ def _check_rp_config(self):
             if clusterConfig[k] != v and "{}".format(clusterConfig[k]) != v:
                 assert False
 
+    def _check_aws_nodes(self):
+        cmd = self.redpanda.kubectl._ssh_prefix() + [
+            'aws', 'ec2', 'describe-instances',
+            '--filters="Name=tag:Name, Values=redpanda-{}-rp"'.format(
+                self._clusterId),
+            '--query="Reservations[0].Instances[*].InstanceType"'
+        ]
+        res = subprocess.check_output(cmd)
+        resd = json.loads(res)
+
+        self.logger.debug(
+            "asserting nodes_count: expected: {}, actual: {}".format(
+                self._configProfile['nodes_count'], len(resd)))
+        assert len(resd) == self._configProfile['nodes_count']
+
+        self.logger.debug(
+            "asserting machineType: expected: {}, actual: {}".format(
+                self._configProfile['machine_type'], resd[0]))
+        assert resd[0] == self._configProfile['machine_type']
+
+    def _check_azure_nodes(self):
+        # currently, we have to override the PATH for azure because
+        # az-cli is installed via snap and /snap/bin only appears in
+        # $PATH on *interactive* shells
+        cmd = self.redpanda.kubectl._ssh_prefix() + [
+            'env', 'PATH=/usr/bin:/bin:/snap/bin',
+            'az', 'aks', 'nodepool', 'list',
+            '--cluster-name', f'aks-rpcloud-{self._clusterId}',
+            '--resource-group', f'rg-rpcloud-{self._clusterId}',
+            '--query', "'[?starts_with(name,`redpanda`)].vmSize'",
+            '--output', 'json'
+        ] # yapf: disable
+
+        res = subprocess.check_output(cmd)
+        resd = json.loads(res)
+
+        self.logger.debug(
```

Comment:
This is a good debug message but you should it as the assert so it's part of the assert message, which gets surfaced much more visibly (e.g., right in report.html).